Automattic/jetpack
# Protect: fix visual height issue of the Interstitial page

This PR fixes a visual issue that happens in the Interstitial page of the Protect plugin, defining the same height for both sections.

Fixes #

#### Changes proposed in this Pull Request:

* Protect: Set same height (100%) of both sections of the Interstitial page

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to Protect interstitial page
* Confirm the height of both sections are the same

before | after
------|------
<img width="1076" alt="image" src="https://user-images.githubusercontent.com/77539/175053184-0a865314-a3d9-466a-a997-afb042d8ec8a.png"> | <img width="1077" alt="image" src="https://user-images.githubusercontent.com/77539/175053256-ffaa7ee2-c73b-446d-9dda-d7b441324c25.png">

Pay attention to the height difference, at the top and bottom of both sections.
